src/test/rpc_tests.cpp

```diff
@@ -391,7 +392,7 @@ BOOST_AUTO_TEST_CASE(rpc_insightexplorer)
 
     // During startup of the real system, fInsightExplorer ("-insightexplorer")
     // automatically enables the next three, but not here, must explicitly enable.
-    fExperimentalMode = true;
```

Comment:
If you make `fExperimentalMode` accessible as I suggested, then keep this line (and the identical one below).